### PR TITLE
Feature - Game room model creator

### PR DESCRIPTION
**Pull request by AndresRoc**
**Objective** 
This model aims to create the objects required to make a game room.

**Changes**
The model has been added to the file models.py

**Ideal behavior**
Server running after executing 
``` python manage.py runserver_plus ```

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
+from email.policy import default
```
Comment (tsoisauce) [inline app/models.py]: what is this for?

Comment (AndresRoc) [inline app/models.py]: Is this required?

File: wavelength/settings.py [hunk 2]
```diff
@@ -32,8 +32,8 @@
 DEBUG = int(os.environ.get('DEBUG', default=0))
 
 # validations
-ALLOWED_HOSTS = ['localhost, 127.0.0.1, vast-earth-36970.herokuapp.com']
-CSRF_TRUSTED_ORIGINS = 'https://vast-earth-36970.herokuapp.com'
+ALLOWED_HOSTS = ['*'] #['localhost, 127.0.0.1 , vast-earth-36970.herokuapp.com']
```
Comment (tsoisauce) [inline wavelength/settings.py]: we will temporarly allow this, in the future we will secure this better.  Nothing need to be done

**Review by tsoisauce — COMMENTED** [review]
@AndresRoc please look...this is a test

**Comment by tsoisauce** [comment]
LGTM

**Review by tsoisauce — COMMENTED** [review]
good job!

**Review by tsoisauce — APPROVED** [review]

**Pull request merged**